pages/events/index.vue

```diff
@@ -10,11 +10,37 @@
           </cv-tabs>
         </client-only>
       </div>
+      <div class="event-page__filters-region">
+        <client-only>
+          <cv-multi-select
+            class="event-age__filters-region_dropdown"
+            :theme="'light'"
+            :label="'All locations'"
+            :inline="false"
+            :options="getOptions('regions')"
+            :selection-feedback="'fixed'"
+            @change="options => updateFilters('regionFilters', options)"
```

Comment:
I think `selectedOptions` is more semantic than just `options`. With `options` I think in all the possible options